Source/DafnyCore/AST/SyntaxDeserializer/Generated.cs

```diff
@@ -2073,6 +2073,7 @@ public TraitDecl ReadTraitDecl()
             var parameter3 = ReadList<TypeParameter>(() => ReadTypeParameter());
             var parameter4 = ReadList<MemberDecl>(() => ReadAbstract<MemberDecl>());
             var parameter7 = ReadList<Type>(() => ReadAbstract<Type>());
+            var parameter8 = ReadList<AttributedExpression>(() => ReadAttributedExpression());
```

Comment:
`parameter8` is not used. Is it mean to be or does `ReadAttributedExpression()` have side effects? (Same comment on L2100.)